io.github.rsvzz.csven/io.github.rsvzz.csven.metainfo.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.rsvzz.csven</id>
+  <name>csven</name>
+  <summary>App para deletrear palablas</summary>
+  <description>
+    <p>Esta app permite aprender palabras en ingles mediante el deletreo letra por letra mediente un juego</p>
+  </description>
+  <categories>
+    <category>Education</category>
+  </categories>
+  <releases>
+    <release version="0.0.3" date="2025-03-12"/>
+  </releases>
+</component>
```

Comment:
missing required screenshots.

Reply:
also missing content-rating, url, developer_info or develop_name, project_license, metadata_license and launchable.